### ISSUE TITLE
Head impulse test

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#9791]
**Preferred term label:**
Head impulse test has corrective saccade

**Synonyms**
Head thrust test

**Definition (free text, please give PubMed ID)**
In a patient with nystagmus, The examiner briskly rotates the patient’s head while having the patient fixate on a target. A corrective saccade indicates the patient's eye will move in one direction and quickly return back to neutral position.

Details at https://neurosigns.org/wiki/Head_impulse_test

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Benign paroxysmal position vertigo

**Your nano-attribution (ORCID)**
